### DISCUSSION TITLE
tox: command not found

### DISCUSSION THREAD

**Discussion by lennart-damen** [ymyzk/tox-gh-actions#167] [Q&A] [answered]
Github Actions seems to be unwilling to bow to tox somehow, not sure what I'm doing wrong. Had the same problem before without using tox-gh-actions. The worflow will start, but when it comes to tox, the log reports:

tox: command not found

Currently my workflow looks as follow:

```name: SimpleOR continuous integration

on:
    pull_request:
        branches: [ master ]

jobs:
    test_pull_request:
        runs-on: ubuntu-latest
        strategy:
            max-parallel: 2
            matrix:
                python-version: [ 3.7, 3.8 ]
        steps:
            - uses: actions/checkout@v1
            - name: Setup Python
              uses: actions/setup-python@v2
              with:
                python-version: ${{ matrix.python }}
            - name: Install Tox and any other packages
              run: |
                  python -m pip install --upgrade pip
                  pip install tox tox-gh-actions
            - name: Run tox
              run: tox
```

And my tox.ini:
```[tox]
envlist = py37, py38, flake8, mypy

[travis]
python =
    3.8: py38
    3.7: py37

[testenv:flake8]
description = Linter with flake8
basepython = python
deps = flake8
commands = flake8 simpleor tests

[testenv:mypy]
description = Static type check with mypy
basepython = python
deps = mypy
commands = mypy simpleor tests --ignore-missing-imports

[testenv]
setenv =
    PYTHONPATH = {toxinidir}
deps =
    -r {toxinidir}/requirements.txt
; If you want to make tox run the tests with the same versions, create a
; requirements.txt with the pinned versions and uncomment the following line:
;     -r{toxinidir}/requirements.txt
commands =
    pip install -U pip
    pytest --basetemp={envtmpdir}

[gh-actions]
python =
    3.7: py37
    3.8: py38
```

Running tox locally is working fine (MacOs Catalina 10.15.7, python 3.8.5, tox 3.20.1).

**Comment by ymyzk** [comment]
Hi @lennart-damen, looks like this is an issue of GitHub Actions configurations rather than tox-gh-actions.

Maybe I can help a bit more but I'd like to see your configuration as is because one posted on this issue is missing indentation. Could you share a link to your repository if it's public or share the YAML file as is using triple quotes (```)?

For example,
```yaml
on:
  pull_request:
    branches: [ master ]
```

**Comment by lennart-damen** [comment]
ahh sorry about that, I updated my previous post :)
link to the repo: https://github.com/lennart-damen/simple-or. The workflow is in .github/workflows, the tox file in the base directory.

**Comment by lennart-damen** [comment, marked as answer]
Was able to work it out, thank you for your quick response anyway!
